package.json

```diff
@@ -15,9 +15,14 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "types": "./index.d.ts",
-      "import": "./dist/esm/i18next.js",
-      "require": "./dist/cjs/i18next.js"
+      "import": {
+        "types": "./index.d.mts",
```

Comment:
If the two typing files are functionally identical, couldn't it just copy the root index.d.ts to both `./dist/esm/i18next.d.ts` and `./dist/cjs/i18next.d.ts` in the build script and direct to those here? It'd have minor duplication but the gzipping of the package would resolve most of that

Reply:
Nope, `export default` is incorrect for commonjs package.